### PR TITLE
Update for 24.2.0

### PR DESCRIPTION
**Pull request by Myoldmopar**
A few tweaks this time to update the supported platforms, but basically the same as always.

### CODE REVIEW

File: src/app/downloads/downloads.component.html [hunk 1]
```diff
@@ -16,16 +16,15 @@ <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to st
             <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.windows_main" (click)="trackDownload('windows', latestRelease.windows_main)">Windows (x64)</a>
           </div>
           <div class="col-xs-12 col-sm-4 col-lg-4 mr-1">
-            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_22" (click)="trackDownload('linux', latestRelease.ubuntu_22)">Ubuntu 22.04</a>
+            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_24" (click)="trackDownload('linux', latestRelease.ubuntu_24)">Ubuntu 24.04</a>
```
Comment (Myoldmopar) [inline src/app/downloads/downloads.component.html:19]: - Ubuntu 24.04 moved up to the new primary version
- Pushed Ubuntu 22.04 down as the alternate
- Removed 20.04 entirely

File: src/app/downloads/downloads.component.html [hunk 2]
```diff
@@ -16,16 +16,15 @@ <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to st
...
             <div class="alt-builds">
-              <a class="card-link text-center ml-2" [href]="latestRelease.ubuntu_20" (click)="trackDownload('linux', latestRelease.ubuntu_20)">Ubuntu 20.04</a>
+              <a class="card-link text-center ml-2" [href]="latestRelease.ubuntu_22" (click)="trackDownload('linux', latestRelease.ubuntu_22)">Ubuntu 22.04</a>
             </div>
           </div>
           <div class="col-xs-12 col-sm-2 col-lg-3">
-            <a class="btn btn-default btn-block btn-primary btn-bg-dark mb-1" role="button" [href]="latestRelease.mac12" (click)="trackDownload('mac', latestRelease.mac12)">MacOS 12</a>
+            <a class="btn btn-default btn-block btn-primary btn-bg-dark mb-1" role="button" [href]="latestRelease.mac13_arm" (click)="trackDownload('mac', latestRelease.mac13_arm)">MacOS 13 (ARM)</a>
```
Comment (Myoldmopar) [inline src/app/downloads/downloads.component.html:25]: - ARM is on the mac13 image on GitHub for us, so updated that as the primary
- Mac x86 is on mac12
- Removed mac11

File: src/app/shared/classes/releases.ts [hunk 3]
```diff
@@ -57,15 +57,18 @@ export const releases: ReleaseItem[] = [{
...
-  windows_main: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Windows-x86_64.exe',
-  ubuntu_22: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Linux-Ubuntu22.04-x86_64.run',
-  ubuntu_20: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Linux-Ubuntu20.04-x86_64.run'
+  version: '24.2.0',
+  date: 'September 30, 2024',
+  mac13_arm: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Darwin-macOS13-arm64.dmg',
+  mac12: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Darwin-macOS12.1-x86_64.dmg',
+  windows_main: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Windows-x86_64.exe',
+  ubuntu_22: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Linux-Ubuntu22.04-x86_64.run',
+  ubuntu_24: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Linux-Ubuntu24.04-x86_64.run'
```
Comment (Myoldmopar) [inline src/app/shared/classes/releases.ts:73]: These binaries aren't all posted yet, so the links won't work as I type this, but they should be done within an hour or so.

File: src/app/documentation/documentation.component.html [hunk 4]
```diff
@@ -54,6 +54,7 @@ <h2>New Features Planning</h2>
                   <p class="font-default mt-3">View the prioritized list of <a href="assets/nrel_custom/eplus_files/EnergyPlus_Prioritized_Feature_Requests_FY22.pdf">new feature requests for FY22</a>.</p>
                   <p class="font-default mt-3">View the prioritized list of <a href="assets/nrel_custom/eplus_files/EnergyPlus_Prioritized_Feature_Requests_FY23.pdf">new feature requests for FY23</a>.</p>
                   <p class="font-default mt-3">View the prioritized list of <a href="assets/nrel_custom/eplus_files/EnergyPlus_Prioritized_Feature_Requests_FY24.pdf">new feature requests for FY24</a>.</p>
+                  <p class="font-default mt-3">View the prioritized list of <a href="assets/nrel_custom/eplus_files/EnergyPlus_Prioritized_Feature_Requests_FY25.pdf">new feature requests for FY25</a>.</p>
```
Comment (Myoldmopar) [inline src/app/documentation/documentation.component.html:57]: Added the feature prioritization for FY25.

File: src/app/testing/testing.component.html [hunk 5]
```diff
@@ -9,7 +9,20 @@ <h1 class="page-header">Testing and Validation</h1>
...
+                  <p>Tracking of the speed and the number of lines and functions that are covered by the tests is also included. Most of these tests can be performed locally by each developer on the team so they can provide the best changes prior to committing them to the common repository.</p>
+                  <h2>Current Validation Testing</h2>
+                  <p>An important ongoing part of EnergyPlus development is testing using industry-standard methods as major builds are completed. The goal is to make EnergyPlus as bug-free as possible. Currently, the validation primarily follows ASHRAE Standard 140 and is conducted as part of the <a href=https://openstudio.net>OpenStudio</a> validation using the same standard. The automated validation system can be seen <a href=https://github.com/NREL/BESTEST-GSR>on the BESTEST-GSR repository</a>.  Note that the results are posted under [individual releases]( https://github.com/NREL/BESTEST-GSR/releases) that generally correspond with software releases of OpenStudio and EnergyPlus.</p>
+                  <h2>Earlier Validation Testing</h2>
+                  <p>Historically, two categories of validation tests were conducted:</p>
```
Comment (Myoldmopar) [inline src/app/testing/testing.component.html:25]: This is an update to the testing page to better reflect current testing.  I'm not set up right now to render the whole web site, but this does look OK when I just render it as raw html.

File: src/app/weather/weather-simulation/weather-simulation.component.html [hunk 6]
```diff
@@ -37,7 +37,7 @@ <h1 class="page-header">Weather Data for Simulation</h1>
...
...s. Currently, there are more than 17,000 TMYx locations in more than 250 countries available for free download. In July 2024, new TMYx covering 2009-2023 were released. TMYx for 2007-2021 and 2004-2018 remain on the web site. Solar radiation data for each site has corresponding solar radiation from the ERA5 reanalysis data set. The ERA5 data, courtesy of Oikolab, provides a comprehensive, worldwide gridded solar radiation data set based on satellite data. The older 2004-2018 TMYx files use cloud cover and other variables to determine solar radiation. Other data sets include: TMY3a (US and some others), TMY (US and some others), CWEC 2020 v2 (564 locations), CWEC TDY 2020 (564 locations), and Canada Future Weather (564 locations); AMTUes v2.5 (Uruguay), U.S. Climate Normals (1981-2010, 1991-2020, and 2006-2020 for 600 locations), CTZ (California Climate Zones), CSWD (Chinese Standard Weather Data), BBSR (Germany), NIWA (New Zealand), RMY 2012 (Australia), IMS (Israel), ITMY (Iran), SWERA (multiple countries), INMET and TRY (Brazil), City University of Hong Kong, ISHRAE (India), IMGW (Poland), KISR (Kuwait), JGMY (Japan generated weather), TurTMY (Turkey), and ArgTMY (Argentina).</p>
```
Comment (Myoldmopar) [inline src/app/weather/weather-simulation/weather-simulation.component.html:40]: Some minor tweaks to this climate data source.

**Review by Myoldmopar — COMMENTED** [review]
OK, I think I'm done with any changes.  Let me know if you see any issues!  Thanks!

**Review by kflemin — APPROVED** [review]

**Pull request merged**